File: Python Programming/problems on numbericals/problem79.py
'''
79. Accept N numbers from user and accept one another number as NO , return index of first occurrence of that NO. 
	
	Input :  N :   6 
	
	NO:  66 
	
	Elements : 85 66 3 66 93 88  
	
	Output : 1 
	
	Input :  N :   6 
	
	NO:  12 
	
	Elements : 85 11 3 15 11 111  
	
	Output : -1  
	
	Author : Milind Nalawade
	
	Date : 10/07/2021
	
	Language : Python 3.9.0
'''

#====
#Logic
#====

def ChkNo(arr):
	
	no=int(input(" Enter your number to find "))
	print()
	
	for i in range(len(arr)):
		if arr[i]==no:
			print("Number found {} in list ".format(i))
			print()
			return
	print(" Number not found in the list !!! ")
	print()
#=============
#Entey point function
#=============

def main():
	arr=[]
	size=int(input(" Enter your size of list : "))
	print()
	for i in range(size):
		no=int(input("Enter element no. {} : ".format(i)))
		print()
		arr.append(no)
	ChkNo(arr)
	
#=====
#starter
#=====
if __name__=="__main__":
	main()

File: C Programming/problem on numbericals/problem79.c
/*

79. Accept N numbers from user and accept one another number as NO , return index of first occurrence of that NO. 
	
	Input :  N :   6 
	
	NO:  66 
	
	Elements : 85 66 3 66 93 88  
	
	Output : 1 
	
	Input :  N :   6 
	
	NO:  12 
	
	Elements : 85 11 3 15 11 111  
	
	Output : -1  
	
	Author : Milind Nalawade
	
	Date : 10/07/2021
	
	Language : C programming
	
*/

#include<stdio.h>

//Logic
void Logic(int arr[],int size)
{
	int no=0;
	printf("\n Enter your number NO : ");
	scanf("%d",&no);
	for(int i=0;i<size;i++)
	{
		if(arr[i]==no)
		{
			printf("\n Number found on index number %d ",i);
			printf("\n");
			return;
		}
	}

	printf(" Entered number is not present in the arrray !!");
	printf("\n");
}
//Entry point function
int main()	
{
	int size=0;
	printf("\n Enter size of array : ");
	scanf("%d",&size);
	printf("\n");
	int arr[size];
	
	for(int i=0;i<size;i++)
	{
		printf(" Enter element at index %d : ",i);
		scanf("%d",&arr[i]);
	}
	Logic(arr,size);
	return 0;
}